src/tests/ftest/cart/selftest/cart_selftest_three_node.yaml

```diff
@@ -5,7 +5,7 @@ ENV:
   default:
     #!filter-only : /run/env_CRT_CTX_SHARE_ADDR/no_sep
     #!filter-only : /run/tests/self_np
-    - D_LOG_MASK: "WARN"
+    - D_LOG_MASK: "DEBUG"
```

Comment:
setting this to DEBUG will blow up sizes of generated log files on every run. we should keep it at warn to keep logs small; if a developer notices an issue with a test, they can then resubmit PR and adjust this setting during their investigation

Reply:
6f1f106efe778c0067ddece8327fd03b95a04af6